src/main/java/com/ipiecoles/java/java220/Cadre.java

```diff
@@ -0,0 +1,72 @@
+package com.ipiecoles.java.java220;
+
+import org.joda.time.LocalDate;
+
+import java.util.Objects;
+
+public class Cadre extends  Employe{
+
+    private Double coefficient = 1d;
+
+    public Cadre(){
+    }
+
+    public Cadre(String nom, String prenom, String matricule, LocalDate dateEmbauche, Double salaire, Boolean tempsPartiel, String sexe, Double coefficient) {
+        super(nom, prenom, matricule, dateEmbauche, salaire, tempsPartiel, sexe);
+        this.coefficient = coefficient;
+    }
+
+    public Cadre(Double coefficient){
+        this.coefficient= coefficient;
+    }
+
+
+    public Double getCoefficient() {
+        return coefficient;
+    }
+
+    public void setCoefficient(Double coefficient) {
+        this.coefficient = coefficient;
+    }
+
+    @Override
+    public Double getPrimeAnnuelle() {
+        return Entreprise.primeAnnuelleBase()*coefficient;
+        //LocalDate.now().getYear()*0.5*coefficient
+    }
+
+    @Override
+    public Integer getNbConges(){
+        return Entreprise.NB_CONGES_BASE + coefficient.intValue();
```

Comment:
Sur le principe, utilise plutôt l'appel à la super méthode (même si au final ça revient au même)
```suggestion
        return super.getNbConges() + coefficient.intValue();
```